Tests/SSL/test_data_modules.py

```diff
@@ -203,7 +194,8 @@ def test_combined_data_module() -> None:
                                                  data_dir=str(path_to_test_dataset),
                                                  # 300 images in total in test dataset
                                                  batch_size=60,
-                                                 shuffle=False)
+                                                 shuffle=False,
+                                                 num_workers=0)
```

Comment:
Why are we changing the default num_workers? was this affecting the len issue?

Reply:
This is not affecting the test, but the test runtime - changes runtime from 1min to 400ms :-)